compiler/infra/OMRCfg.cpp

```diff
@@ -719,16 +719,10 @@ TR::CFGNode *OMR::CFG::removeNode(TR::CFGNode *node)
    return node;
    }
 
-enum OrphanType
-   {
-   IsParented = 0,
-   IsOrphanedNode,
-   IsOrphanedRegion
-   };
-
 // test if 'to' is an unreachable orphan
 //
-static OrphanType unreachableOrphan(TR::CFG *cfg, TR::CFGNode *from, TR::CFGNode *to)
+TR::CFG::OrphanType
```

Comment:
This enum isn't extended and lives in the OMR namespace.  Shouldn't it be `OMR::CFG::OrphanType` ?